exporter/context.go

```diff
@@ -351,9 +351,6 @@ func (ic *importContext) Has(r *resource) bool {
 }
 
 func (ic *importContext) Add(r *resource) {
-	if ic.Has(r) {
```

Comment:
this is a short-circuit for not adding duplicate resources. cannot be removed.

Reply:
ah, ok - I see, there could be a case where were nested import will re-add the same resource. I'll re-implement it a bit differently

Reply:
@nfx what about updated implementation?